### DISCUSSION TITLE
Lora not visible in webui forge

### DISCUSSION THREAD

**Discussion by dvelm** [huggingface/autotrain-advanced#527] [Q&A]
I used https://colab.research.google.com/github/huggingface/autotrain-advanced/blob/main/colabs/AutoTrain.ipynb for many lora sd xl and it's visible in fooocus webui but in sd webui forge not visible https://github.com/lllyasviel/stable-diffusion-webui-forge . The only one which is visible is sd_xl_offset_example-lora_1.0.safetensors downloaded from fooocus

**Comment by abhishekkrthakur** [comment]
please create an issue in forge

**Reply by dvelm** [reply]
A question. Is that Colab only for SD XL or also for other models? Trying 1.5 or fine-tuned results errors

**Reply by urostrickovic** [reply]
They also don't show up in Automatic 1111 and i cant activate them. in fooocus they work perfectly.
